src/consensus/tx_verify.h

```diff
@@ -52,7 +52,7 @@ unsigned int GetP2SHSigOpCount(const CTransaction& tx, const CCoinsViewCache& ma
  * @param[out] flags Script verification flags
  * @return Total signature operation count for a tx
  */
-unsigned int GetTransactionSigOpCount(const CTransaction& tx, const CCoinsViewCache& inputs, int flags);
+unsigned int GetTransactionSigOpCount(const CTransaction& tx, const CCoinsViewCache& inputs, uint32_t flags);
```

Comment:
22232: missing https://github.com/bitcoin/bitcoin/pull/22232/files#diff-fbcf485bde2750c24663197937e09a3974208ebb5a4cd2c03d4168d7e3ff195bR52

also missing changes in tests, so should be `partial` too